en/setup/dev_env_mac.md

```diff
@@ -1,6 +1,11 @@
 # Development Environment on Mac
 
-Mac OS X is the main development platform for PX4. The following instructions explain how to set up a development environment for building NuttX-based hardware (Pixhawk, etc.) and Simulation (jMAVSim/Gazebo8) targets. For other targets see: [Toolchain Installation > Supported Targets](../setup/dev_env.md#supported-targets).
+Mac OS X is the main development platform for PX4. The following instructions set up a development environment for building:
```

Comment:
> Mac OS X is the main development platform for PX4

This seems a bit contradictory to the statement on the "Toolchain Installation" page:
> We recommend Ubuntu Linux LTS edition as this enables building all PX4 targets, and using most simulators and ROS.

What makes Mac the main dev platform / what is the reader supposed to gather from this statement?

Reply:
Yes, I'm aware. The statement predates me. I'm not sure on what it was based. @bkueng What would you say defines the "main dev platform". I've always though Linux makes sense as a free OS that builds more targets, but you could argue that MacOS fits under criteria of "easiest setup".

Reply:
Fair enough. I have fixed this in https://github.com/PX4/Devguide/pull/554 - we have no "main" dev env, just ones that we support.